tests/test_analyze.py

```diff
@@ -135,3 +135,15 @@ def test_analyze_for_different_detectors():
                     assert result["gender"]["Man"] > result["gender"]["Woman"]
                 else:
                     assert result["gender"]["Man"] < result["gender"]["Woman"]
+
+def test_analyze_for_multiple_faces():
+    img = "dataset/img4.jpg"
+    # Copy and combine the same image to create multiple faces
+    img = cv2.imread(img)
+    img = cv2.hconcat([img, img])
+    demography_objs = DeepFace.analyze(img, silent=True)
```

Comment:
you should check the length of demography_objs, it should be 2 because 2 images fed (i hope only one face detected in img4.jpg)